### ISSUE TITLE
Update Bootstrap to v5.3 

### ISSUE THREAD

**Issue by visto9259** [laminas/laminas-mvc-skeleton#84]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | no
| RFC         |  no
| BC Break    |  no

#### Summary

The version of Bootstrap that is bundled in the skeleton is version 5.2.  Bootstrap has been on 5.3 for a year now.

The skeleton should use the bundled Bootstrap JS to have all the tools.

It may be preferable to use the CDN?

**Comment by visto9259** [comment]
If you agree to use the bundled Bootstrap js file, I have the pull request ready to submit

**Comment by gsteel** [comment]
Send in the patch @visto9259 👍

**Comment by juizmill** [comment]
I think there should be a package manager like webpack, vite etc.

Making it much easier and more modern when using CSS and JS packages.

**Comment by gsteel** [comment]
> I think there should be a package manager like webpack, vite etc.
> 
> Making it much easier and more modern when using CSS and JS packages.

Not really. Everyone does front-end differently. It is not appropriate for a PHP project skeleton to dictate which front-end tech to use.

It's enough to provide pre-compiled Bootstrap so that it can easily be ripped out and replaced with whatever the user wants.

**Comment by juizmill** [comment]
> > I think there should be a package manager like webpack, vite etc.
> > Making it much easier and more modern when using CSS and JS packages.
> 
> Not really. Everyone does front-end differently. It is not appropriate for a PHP project skeleton to dictate which front-end tech to use.
> 
> It's enough to provide pre-compiled Bootstrap so that it can easily be ripped out and replaced with whatever the user wants.

My suggestion is to create a skeleton for the front-end base with modern tools. Currently Bootstrap is set to "default"

**Comment by visto9259** [comment]
@juizmill 
I understand your need to have a "starter kit" that provides a framework for front end work.
Unfortunately, it will not come from this project. This skeleton's purpose is to provide a quick starting point for the development of a PHP server using the Laminas MVC and component framework.
The skeleton does not "endorse" Bootstrap as the front end CSS and JS to use in applications. Bootstrap is just an example to get users started quickly.

I personally use Bootstrap and React in my app based on Laminas MVC. I use Grunt, Webpack and other tools in the dev/build workflows. 

Take a look at this guide https://visto9259.github.io/docs/laminas-webpack-guide-getting-started

I will not pretend that it is the best workflow as it is outdated and could be improved but it will give you a starting point for what you are looking for.

**Comment by thalles** [comment]
I think CDN is a good way
